src/wallet/wallet2.h

```diff
@@ -648,6 +650,12 @@ namespace tools
         FIELD(dests)
         FIELD(construction_data)
         FIELD(multisig_sigs)
+        if (version < 1)
+        {
+          multisig_tx_key_entropy = crypto::null_skey;
+          return true;
+        }
+        FIELD(multisig_tx_key_entropy)
```

Comment:
Didn't 8149 simply ban old objects when deserializing? Why not do the same here?